ocaml/gencert/selfcert.ml

```diff
@@ -137,13 +137,18 @@ let host ~name ~dns_names ~ips pemfile =
   in
   R.failwith_error_msg res
 
+let serial_stamp () = Unix.gettimeofday () |> string_of_float
```

Comment:
Using floats here seems fine, in the RFCs I couldn't find any requirement for the field relating to the issuer, and for the subject only uniqueness is required per CA:

> The serialNumber attribute type SHALL, when present, be used to
      differentiate between names where the subject field would
      otherwise be identical.  This attribute has no defined semantics
      beyond ensuring uniqueness of subject names.  It MAY contain a
      number or code assigned by the CA or an identifier assigned by a
      government or civil authority.  It is the CA's responsibility to
      ensure that the serialNumber is sufficient to resolve any subject
      name collisions.